modules/data.atmosphere/R/extract_local_CMIP5.R

```diff
@@ -153,10 +173,34 @@ extract.local.CMIP5 <- function(outfolder, in.path, start_date, end_date, site_i
       lat_bnd <- ncdf4::ncvar_get(ncT, "lat_bnds")
       lon_bnd <- ncdf4::ncvar_get(ncT, "lon_bnds")
       nc.time <- ncdf4::ncvar_get(ncT, "time")
-      
+
       # splt.ind <- ifelse(GCM %in% c("MPI-ESM-P"), 4, 3)
       # date.origin <- as.Date(str_split(ncT$dim$time$units, " ")[[1]][splt.ind])
- 
+      nc.date <- date.origin + nc.time
+      date.leaps <- seq(as.Date(paste0(files.var[[var.now]][i,"first.year"], "-01-01")), as.Date(paste0(files.var[[var.now]][i,"last.year"], "-12-31")), by="day")
+      # Figure out if we're missing leap dat
+      no.leap <- ifelse(is.null(no.leap) & length(nc.date)!=length(date.leaps), TRUE, FALSE)
```

Comment:
FWIW, the ifelse seems unnecessary here -- is there a reason you couldn't rewrite this as:
```
no.leap <- is.null(no.leap) & length(nc.date)!=length(date.leaps)
```

Also, be aware that @ashiklom rewrote/standardized a lot of the leap processing code throughout the rest of PEcAn.  Alexey: (1) could you check Christy's code to see if your functions would be useful here as a later PR, (2) we should probably add those functions to the "code standards" part of the documentation, else we'll all keep reinventing them